docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,259 @@
+
+> Whether or not an error occurs when it is opened, resources associated with the database connection handle should be released by passing it to sqlite3_close() when it is no longer required.
+
+Rarely this happens, but if the `sqlite3_close` **fails to be executed**, we will be just killing the bot's process. Since this is not a highly detailed guide, we won't be teaching how to deal with it with many details, but you will only need to finish the finalize the ongoing statement and then use `sqlite3_close` again.
+
+### Creating a sqlite3 table
+
+Firstly, before trying to save data into the DB file, we will need to create a sqlite3 table first. sqlite3 tables contain a schema for that DB. The schema for a DB is "a description of all of the other tables, indexes, triggers, and views that are contained within the database".
+
+```c
+char *msgErr = NULL;
+char *query = sqlite3_mprintf("CREATE TABLE IF NOT EXISTS concord_guides(user_id INT, guild_id INT);");
+rc = sqlite3_exec(db, query, NULL, NULL, &msgErr);
+
+sqlite3_free(query);
+
+if (rc != SQLITE_OK) {
+    log_fatal("[SQLITE] Something went wrong while creating concord_guides table. [%s]", msgErr);
+    if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+    return;
+}
+
+sqlite3_close(db);
+```
+
+Here, we are defining the command that **will be executed** with `sqlite3_mprintf`, the command going to be executed is "`CREATE TABLE IF NOT EXISTS concord_guides(user_id INT, guild_id INT)`", where **if concord_guides table doesn't exist**, it is going to be created.
+
+Also, we are saving **INT values**, but if you wanted to **save a text TEXT value**, you can simply replace the `INT` with `TEXT`. (WARN: **This will cause you to do some modifications when retrieving data**)
+
+Then `sqlite3_exec` is going to perform the command defined with `sqlite3_mprintf`, and, **if some error happens**, it is going to write on the `char *msgErr`. After `sqlite3_exec`, we are going to free the query (generated by `sqlite3_mprintf`) to **not cause a memory leak**.
+
+After freeing the query, we **check if the status code of the `sqlite3_exec` is not equal to `SQLITE_OK`**, in case it's not, it's going to log a fatal saying that it failed that something failed while creating the table, and also includes the error message and closing the DB.
+
+Done, if nothing wrong happens, now you've created a table called concord_guides.
```

Comment:
"Done! If nothing wrong happens, you've created a table called concord_guides."

Reply:
Thanks:)